manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -613,3 +613,8 @@ properties:
 
   garden:
     log_level: error
+  racoon:
+    certificate_authority_cert: (( grab secrets.bosh_ca_cert ))
+    certificate_authority_private_key: (( grab secrets.bosh_ca_key ))
```

Comment:
> we should do that for any other private key that we've got currently in our manifest.

Could you clarify what you mean by "do that"? We do currently generate those other private keys upfront and provide them as properties. Although I suspect there may be other examples of world-readable files.

The reason it's a particular problem to distribute the BOSH CA this way is that, should somebody compromise a machine that's using IPsec, they can go on to sign new certificates that will be trusted by the rest of the machines, e.g. impersonate the SSH proxy. Whereas if one of the individual certificates is leaked then the damaged is limited to that one service.